crates/rspack_plugin_javascript/src/plugin/impl_plugin_for_js_plugin.rs

```diff
@@ -284,13 +284,13 @@ impl Plugin for JsPlugin {
         .runtime(&chunk.runtime),
     );
     asset_info.set_javascript_module(compilation.options.output.module);
-    Ok(vec![RenderManifestEntry::new(
+    Ok(TWithDiagnosticArray::from(vec![RenderManifestEntry::new(
```

Comment:
Can we just import trait `IntoTWithDiagnosticArray` and use `with_empty_diagnostic`? `From` implementation seems not that necessary to me.

Reply:
Wow, didn't notice that before, done